### ISSUE TITLE
请问实验结果是怎样选取的？

### ISSUE THREAD

**Issue by t2413** [liuwei1206/LEBERT#61]
是最后一个epoch保存的模型在测试集上的结果、还是dev集上达到最优时在test集上的结果、还是所有epoch中测试集上的最优结果？

**Comment by liuwei1206** [comment]
on dev

**Comment by iamqiz** [comment]
我看了代码 (Trainer.py) 里面在每个epoch结束和训练结束时保存模型,并没有保存dev集最优模型 ,啥情况呀?   @liuwei1206

**Comment by minskiter** [comment]
> on dev

The model was only trained for 20 epochs. What does `on dev` means? Does it mean the best result on the validation set?
